estela-api/builds/config.py

```diff
@@ -0,0 +1,10 @@
+from builds.default import DefaultBuild
+from builds.gcp import GCPBuild
+
+def Builds(type):
```

Comment:
We should add a section in the documentation mentioning what is the `Build` class